fury/molecular.py

```diff
@@ -0,0 +1,729 @@
+
+    def get_atomic_number(self, element_name):
+        """Given a case-insensitive string that contains the symbol or name of
+        an element, return the corresponding atomic number.
+
+        Parameters
+        ----------
+        element_name : string
+            Name of the element whose atomic number is to be obtained.
+        """
+        return self.GetAtomicNumber(element_name)
+
+    def get_atomic_radius(self, atomic_number, radius_type):
+        """Given an atomic number, return either the covalent radius of the
+        atom or return the Van Der Waals radius of the atom depending on
+        radius_type.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose covalent radius is to be
+            obtained.
+        radius_type : string
+            Two valid choices -
+            * 'VDW' : for Van Der Waals radius of the atom
+            * 'Covalent' : for covalent radius of the atom
+        """
+        if radius_type == 'VDW':
+            return self.GetVDWRadius(atomic_number)
+        elif radius_type == 'Covalent':
+            return self.GetCovalentRadius(atomic_number)
+
+    def get_atom_color(self, atomic_number):
+        """Given an atomic number, return the RGB tuple associated with that
+        element (CPK coloring convention) provided by the Blue Obelisk Data
+        Repository.
+
+        Parameters
+        ----------
+        atomicNumber : int
+            Atomic number of the element whose RGB tuple is to be obtained.
+        """
+        return self.GetDefaultRGBTuple(atomic_number)
+
+
+def make_molecularviz_aesthetic(molecule_actor):
+    """Manipulating the lighting to make the molecular visualization
+    aesthetically pleasant to see.
+
+    Parameters
+    ----------
+    molecule_actor : vtkActor
+        Actor that represents the molecule to be visualized.
+    """
+    molecule_actor.GetProperty().SetDiffuse(1)
+    molecule_actor.GetProperty().SetSpecular(0.5)
+    molecule_actor.GetProperty().SetSpecularPower(90.0)
+
+
+def molecular_sphere_rep_actor(molecule, colormode='discrete'):
+    """Create an actor for sphere molecular representation. It's also referred
+    to as CPK model and space-filling model.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to be rendered.
+    colormode : string
+        Set the colormode for coloring the atoms. Two valid color modes -
+        * 'discrete': Atoms are colored using CPK coloring convention.
+        * 'single': All atoms are colored with same color(grey).
+
+        RGB tuple used for coloring the atoms when 'single' colormode is
+        selected: (150, 150, 150)
+        Default is 'discrete'.
+
+    Returns
+    -------
+    molecule_actor : vtkActor
+        Actor created to render the space filling representation of the
+        molecule to be visualized.
+    """
+    msp_mapper = MoleculeMapper()
```

Comment:
@skoudoro  the MoleculeMapper class is used here to create the mapper for sphere representation actor.